docs/application-connector/docs/08-10-rotate-root-ca.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Rotate the Root CA certificate and key
+type: Tutorials
+---
+
+The Central Connector Service uses the Root CA certificate to issue new certificates for runtimes and by the Nginx Ingress Controller to validate their identity.
+
+Two different components use the Root CA certificate. As a result, the certificate is stored in two separate Secrets:
+  - The `connector-service-app-ca` Connector Service CA Secret responsible for signing certificate requests
+  - The `nginx-auth-ca` Nginx Ingress Secret responsible for security in the Connector Service API
+
+Keeping both secrets up-to-date is vital for the security of your implementation as it guarantees that the Connector Service issues proper certificates and no unregistered applications can access its API.
+
+The Root CA certificate has a set expiration date and must be renewed periodically to prevent its expiration. You must also renew the Root CA certificate and key every time they are compromised.
+
+This tutorial describes the procedure you must follow for these scenarios:
+  - Rotating a soon-to-expire Root CA certificate
+  - Rotating a compromised Root CA certificate
+  - Rotating a compromised Root CA key
+
+## Rotating a soon-to-expire CA certificate
+
+To successfully rotate a soon-to-expire CA certificate, you must replace it with a new certificate in both the Connector Service CA Secret and the Nginx Ingress Secret. Follow these steps to replace the old certificate in both Secrets:
+
+1. Get the existing Root CA key. Fetch it from the `connector-service-app-ca` Secret and save it to a `ca.key` file. Run:
+  ```
+  kubectl -n kyma-integration get secret connector-service-app-ca -o=jsonpath='{.data.ca\.key}' | base64 -D > ca.key
+  ```
+
+2. Generate a new certificate for the key you obtained and save it to a `new-ca.crt` file. Run:
+>**NOTE:** Use the `-days` flag to set the TTL of the newly generated certificate.
```

Comment:
Shouldn't we place the NOTE under the command? Somehow it disturbs the flow while reading.